packages/react-player/.npmrc

```diff
@@ -0,0 +1,2 @@
+@lottiefiles:registry=https://npm.pkg.github.com
+@dotlottie:registry=https://npm.pkg.github.com
```

Comment:
why do we need the `@dotlottie` scope and githhub npm registry ? are we installing any private packages from `@dotlottie` org ?

Reply:
oh yeah. only using @lottiefiles. Removing it.